crml/cennzx-spot/src/lib.rs

```diff
@@ -205,21 +205,15 @@ decl_module! {
 				Error::<T>::TradeAssetBalanceToAddLiquidityTooLow
 			);
 			let exchange_key = (core_asset_id, asset_id);
-			let total_liquidity = <TotalSupply<T>>::get(&exchange_key);
+			let total_liquidity = <TotalLiquidity<T>>::get(&exchange_key);
 			let exchange_address = T::ExchangeAddressGenerator::exchange_address_for(core_asset_id, asset_id);
```

Comment:
needs a rebase?

Reply:
Yes, we will need a rebase - I will do the rebase once #183 is merged

Reply:
#183 is done, rebase is done 😄